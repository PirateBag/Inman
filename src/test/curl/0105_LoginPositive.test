Add all possible items in a single batch.

===  JSON  ===
{ "userName" : "fred",
  "password" : "dilban"
}
