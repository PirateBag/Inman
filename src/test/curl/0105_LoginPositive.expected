{
  "token" : "78UIjk",
  "status" : "78UIjk"
}